ScoutSuite/output/data/html/partials/gcp/services.computeengine.projects.id.regions.id.subnetworks.html

```diff
@@ -14,6 +14,7 @@ <h4 class="list-group-item-heading">Information</h4>
         <div class="list-group-item-text item-margin">IP Range: <span id="computeengine.projects.{{@../../key}}.regions.{{@../key}}.subnetworks.{{@key}}.ip_range"><samp>{{ip_range}}</samp></span></div>
         <div class="list-group-item-text item-margin">Gateway Address: <span id="computeengine.projects.{{@../../key}}.regions.{{@../key}}.subnetworks.{{@key}}.gateway_address"><samp>{{gateway_address}}</samp></span></div>
         <div class="list-group-item-text item-margin">Private Google Access: <span id="computeengine.projects.{{@../../key}}.regions.{{@../key}}.subnetworks.{{@key}}.private_ip_google_access_disabled">{{convert_bool_to_enabled private_ip_google_access}}</span></div>
+        <div class="list-group-item-text item-margin">VPC Flow Logs: <span id="computeengine.projects.{{@../../key}}.regions.{{@../key}}.subnetworks.{{@key}}.flowlogs_enabled">{{flowlogs_enabled}}</span></div>
```

Comment:
Add convert_bool_to_enabled (see Private Google Access: just above) in order to make the value as Enabled/Disabled instead of True/False

Reply:
I'm not too confident about this, since there is a tertiary "UNKNOWN" variable as of now due to the indeterminate nature of whether the GCP API displays flow log status. Can I keep it in its current state until the upstream api displays this for all calls?

Reply:
Alright